src/tokens/TokenBalances.tsx

```diff
@@ -38,39 +38,26 @@ function TokenBalancesScreen({ navigation }: Props) {
   const { t } = useTranslation()
   const tokens = useSelector(tokensWithTokenBalanceSelector)
   const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
-  const totalBalance = useSelector(totalTokenBalanceSelector)
+  const totalBalance = useSelector(totalTokenBalanceSelector) ?? new BigNumber(0)
   const tokensAreStale = useSelector(stalePriceSelector)
   const showPriceChangeIndicatorInBalances = useSelector(showPriceChangeIndicatorInBalancesSelector)
   const shouldVisualizeNFTsInHomeAssetsPage = useSelector(
     visualizeNFTsEnabledInHomeAssetsPageSelector
   )
   const walletAddress = useSelector(walletAddressSelector)
 
-  const header = () => {
-    return (
-      <View style={styles.header}>
-        <Text style={fontStyles.navigationHeader}>{t('balances')}</Text>
-        <Text style={styles.subtext}>
-          {tokensAreStale ? (
-            <Text>
-              {localCurrencySymbol}
-              {'-'}
-            </Text>
-          ) : (
-            totalBalance &&
-            t('totalBalanceWithLocalCurrencySymbol', {
-              localCurrencySymbol,
-              totalBalance: totalBalance.toFormat(2),
-            })
-          )}
-        </Text>
-      </View>
-    )
-  }
-
   useLayoutEffect(() => {
+    const subTitle = tokensAreStale
```

Comment:
I wonder, why are we displaying `"${localCurrencySymbol} -"` or `"-"`. Should we use the same one?

Like:
```
const subTitle = !tokensAreStale && totalBalance.gte(0) ?
    t('totalBalanceWithLocalCurrencySymbol', {
          localCurrencySymbol,
          totalBalance: totalBalance.toFormat(2),
        })
    : '-'
```

Reply:
Adjusted in 58369b48a19dad5e693d8048c6a734d336a80b87!